docroot/modules/custom/layout_builder_custom/css/layout_builder_custom.overrides.css

```diff
@@ -127,6 +127,99 @@
   width: auto;
 }
 
+#drupal-off-canvas-wrapper .form [type=checkbox],
+#drupal-off-canvas-wrapper .form [type=radio] {
+  height: 1.2rem;
+  min-width: 1.2rem;
+  margin: .25rem .35rem 0 .25rem;
+  -webkit-transform: scale(1.3, 1.3);
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+/**
+ * Banner block form styles.
+ *
+ * BEM Hierarchy:
+ *
+ * block-form-group (Block)
+ * ├── block-form-group__container (Element - main grouping wrapper)
+ * └── block-form-group__collapsible (Element - details/accordion)
+ *     └── block-form-group__collapsible--overlay (Modifier - gradient/overlay variant)
+ */
+
+/* Remove border and padding for main radio buttons within the form group containers. */
+#drupal-off-canvas-wrapper fieldset {
+  border: none !important;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+}
+
+#drupal-off-canvas-wrapper fieldset:not([data-drupal-ck-style-fence]) {
+  margin-block-end: 0;
+}
+
+/* Block form group container - creates visual grouping */
+#drupal-off-canvas-wrapper .block-form-group__container {
```

Comment:
Wondering if we want to change the naming of this `block-form-group__container` class to be something more generic for off-canvas? It seems like more of an off-canvas forms thing than block form specific.

Reply:
If you think it will have some utility outside of form styling, I think that works. Otherwise, maybe `off-canvas-form-group`?